src/common/load-plugins.js

```diff
@@ -18,10 +18,9 @@ function loadPlugins(plugins, pluginSearchDirs) {
   }
   // unless pluginSearchDirs are provided, auto-load plugins from node_modules that are parent to Prettier
   if (!pluginSearchDirs.length) {
-    const autoLoadDir = thirdParty.findParentDir(
-      thirdParty.findParentDir(__dirname, "prettier"),
-      "node_modules"
```

Comment:
Actually, why don't we just find the parent `"node_modules"`? I don't think it's possible that there's any `"node_modules"` between `__dirname` and `"prettier"`.

cc @kachkaev

Reply:
that does seem to work for me as well, (verified by editing the installed files to this in a nodeenv with `npm install -g`d `prettier`)